src/JuMP.jl

```diff
@@ -836,6 +922,8 @@ end
 
 ##########################################################################
 # Operator overloads
+typealias JuMPTypes Union(Variable,Norm,AffExpr,QuadExpr,SOCExpr)
```

Comment:
Fairly worried, but without justification, what the implications this has for extensions D: